src/components/tweetCard.vue

```diff
@@ -1,112 +1,99 @@
+                  {{ tag }}
+              </p>
+              <p class="text-grey text-sm ml-1">· {{ timeAgoDate(tweet?.updatedAt) }}</p>
+              <button v-show="isAuth" v-if="userName === user?.name" @click="deleteTweet(tweet?._id)"
+                  class="p-1 text-sm text-darkblue hover:text-grey">
+                  <p>· Delete</p>
+              </button>
+          </div>
 
-      <p class="text-left py-3">{{ tweet.text }}</p>
-      <div class="flex items-center place-content-end text-md px-4 text-grey">
-        <button
-          v-show="isAuth"
-          @click="addLike(tweet._id)"
-          class="flex items-center place-content-end hover:text-lightblue"
-        >
-          <font-awesome-icon icon="fa-regular fa-heart" class="mr-3" />
-          <p>{{ tweet.likes.length }}</p>
-        </button>
-        <div v-show="!isAuth" class="flex items-center place-content-end">
-          <font-awesome-icon icon="fa-regular fa-heart" class="mr-3" />
-          <p>{{ tweet.likes.length }}</p>
-        </div>
+          <p class="text-left py-3">{{ tweet?.text }}</p>
+          <div class="flex items-center place-content-end text-md px-4 text-grey">
+            <button
+              v-show="isAuth"
+              @click="addLike(tweet._id)"
+              class="flex items-center place-content-end hover:text-lightblue"
+            >
+              <font-awesome-icon icon="fa-regular fa-heart" class="mr-3" />
+              <p>{{ tweet.likes.length }}</p>
+            </button>
+            <div v-show="!isAuth" class="flex items-center place-content-end">
+              <font-awesome-icon icon="fa-regular fa-heart" class="mr-3" />
+              <p>{{ tweet.likes.length }}</p>
+            </div>  
+          </div>
       </div>
-    </div>
   </div>
 </template>
 
 <script lang="ts">
-import { defineComponent, PropType, ref, computed } from "vue";
-import { useTweetsStore } from "@/store";
-import { useUsersStore } from "@/store/user";
-import Tweet from "@/interfaces/Tweets";
-import moment from "moment";
-import { useRouter } from "vue-router";
+import { defineComponent, PropType, ref, computed } from 'vue'
+import { useTweetsStore } from '@/store';
+import { useUsersStore } from '@/store/user';
+import Tweet from '@/interfaces/Tweets';
+import moment from 'moment'
+import { useRouter } from 'vue-router'
 
 export default defineComponent({
   name: "tweetCard",
   props: {
-    tweet: {
-      type: Object as PropType<Tweet>,
-      required: true,
-    },
-    avatar: {
-      type: Object as PropType<string | undefined>,
-      required: true,
-    },
-    name: {
-      type: Object as PropType<string | undefined>,
-      required: true,
-    },
+      tweet: {
+          type: Object as PropType<Tweet>,
+          required: true,
+      },
   },
   setup() {
-    const isAuth = ref(true);
-    const tweetsStore = useTweetsStore();
-    const router = useRouter();
```

Comment:
sound like unecessary changes. please use prettier to format code as it was used in the repository.